r-package/grf/tests/testthat/test_regression_forest.R

```diff
@@ -158,3 +158,44 @@ test_that("linear correction variables function as expected", {
 
     expect_true(mse.selected < mse / 1.5)
 })
+
+test_that("local linear tuning always returns lambda", {
+   n = 100
+   p = 5
+   sigma = 1
+
+   mu = function(x){log(1+exp(6*x[1]))}
+
+   X = matrix(runif(n*p,-1,1), nrow = n)
+   Y = apply(X, FUN = mu, MARGIN = 1) + sigma*rnorm(n)
+
+   num.reps = 100
+   lambda.indicators = replicate(num.reps, {
+      forest = regression_forest(X, Y, num.trees = 200)
+      lambda = tune_local_linear_forest(forest)$lambda.min
+      is.numeric(lambda)
```

Comment:
Similar to my comment above, I don't think that this will check that `lambda` is a scalar, but only that it's a vector of numbers.